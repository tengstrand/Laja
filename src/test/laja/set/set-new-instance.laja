#write "target/test-output/set-new-instance.out"
  ## "java.lang." is automatically added to the class name if the class is not found in empty package.
  #set (hello1 = new String("Hello"))
  #set (hello2 = new java.lang.String("Hello again!"))
  #set (createme = new net.sf.laja.integrationtest.SetTest$CreateMe("Hello", { name: "Bill" length: 175 }))
hello1 = {hello1}
hello2 = {hello2}
createme = {createme}
#end