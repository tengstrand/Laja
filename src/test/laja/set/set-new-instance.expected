hello1 = Hello
hello2 = Hello again!
createme = Message=Hello, map={name=Bill, length=175}
